<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Postmortem</class>
 <widget class="QMainWindow" name="Postmortem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1606</width>
    <height>978</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Postmortem</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>580</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ltPostmortemHCPlot"/>
   </widget>
   <widget class="QwtPlot" name="pltRf">
    <property name="geometry">
     <rect>
      <x>556</x>
      <y>640</y>
      <width>440</width>
      <height>240</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>620</y>
      <width>150</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Delta RF Postmortem</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>780</width>
      <height>580</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ltPostmortemVCPlot"/>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>600</y>
      <width>220</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertical Correctors Postmortem</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>600</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Horizontal Correctors Postmortem</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>890</y>
      <width>36</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>537</x>
      <y>737</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hz</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="footer">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>920</y>
      <width>1580</width>
      <height>48</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid black;
    border-radius: 5px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btnDialog">
       <property name="text">
        <string>No File Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSelectedFile">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>307</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
